src/containers/ScoreContainer/ScoreContainer.jsx

```diff
@@ -122,8 +122,8 @@ class ScoreContainer extends Component {
             {uri && (
               <Score
                 uri={uri}
-                key={"score" + fullScreen}
-                options={this.OPTIONS}
+                key={'score' + fullScreen}
```

Comment:
Minor: convention to use [template literals](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals) instead of plus